Source/SwiftLintCore/Models/Command.swift

```diff
@@ -85,21 +84,20 @@ public struct Command: Equatable {
     ///
     /// - parameter actionString: The string in the command's definition describing its action.
     /// - parameter line:         The line in the source file where this command is defined.
-    /// - parameter character:    The character offset within the line in the source file where this command is
-    ///                           defined.
-    public init(actionString: String, line: Int, character: Int) {
+    /// - parameter range:        The range of the command in the line (0-based).
+    public init(actionString: String, line: Int, range: Range<Int>) {
```

Comment:
I renamed the parameter to `commandString` in my branch, which I think is better - the comment is also wrong here - it's the local `actionString` on line 99/97 that contains the action.